go/chat/boxer.go

```diff
@@ -219,6 +222,10 @@ func (b *Boxer) boxMessageV1(ctx context.Context, msg chat1.MessagePlaintextV1,
 		}
 	}
 
+	if len(msg.ClientHeader.TlfName) == 0 {
```

Comment:
Writing a blank name is pretty bad, so I think I'll leave this guard in, even though KBFS currently should always return a value.